my-react-app/src/views/customer/MenuPage.tsx

```diff
@@ -14,35 +15,105 @@ const optionButtonStyle = {
 {
   /*These constant variables should be instaciated from the database, for now these will be filled up based from defaults that I will set*/
 }
-const toppingOption_1 = "Fudge";
-const toppingOption_2 = "M&Ms";
-const toppingOption_3 = "Oreos";
-
-const flavor_1 = "Strawberry";
-const flavor_2 = "Vanilla";
-const flavor_3 = "Chocolate";
-
-const cone_1 = "Cake";
-const cone_2 = "Sugar";
-const cone_3 = "Waffle";
+const toppingsList = ["Fudge", "M&Ms", "Oreos"];
+const flavorList = ["Strawberry","Vanilla","Chocolate"];
+const conesList = ["Cake","Sugar","Waffle"];
```

Comment:
These should be pulled from the store, which shouldn't be too hard.  I could be wrong, but I think something like `const toppingsList = useStore((state) => state.products.filter((product) => product.type === ProductType.TOPPING)).map((product) => product.name);`  which would just return a list of names like the hard coded one here.  At some point before confirming the order we'll need to connect the Product ID to the FullCone along with the name and type at the very least, but maybe that can be done in the Cart Page.